stats/stats/src/update/new_blocks.rs

```diff
@@ -0,0 +1,389 @@
+use async_trait::async_trait;
+use chrono::NaiveDate;
+use entity::{chart_data_int, charts};
+use sea_orm::{
+    sea_query, ColumnTrait, DatabaseConnection, DbBackend, EntityTrait, FromQueryResult,
+    QueryFilter, QuerySelect, Set, Statement,
+};
+
+use super::UpdateError;
+
+#[derive(FromQueryResult)]
+struct NewBlocksData {
+    day: NaiveDate,
+    count: i64,
+}
+
+#[derive(Debug, FromQueryResult)]
+struct ChartData {
+    id: i32,
+    date: NaiveDate,
+}
+
+#[derive(Debug, FromQueryResult)]
+struct ChartId {
+    id: i32,
+}
+
+#[derive(Default, Debug)]
+pub struct Updater {}
+
+#[async_trait]
+impl super::UpdaterTrait for Updater {
+    async fn update(
+        &self,
+        db: &DatabaseConnection,
+        blockscout: &DatabaseConnection,
```

Comment:
Nit: maybe name that `blockscout_db` to make it more explicit

Reply:
the type kinda hints that, no?..
`blockscout: Db`

Reply:
Personally, I prefer `blockscout_db` as it seems to be more clear that it is database connection without even looking on the function arguments. But it is probably just a question of preferences